thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/current-wow/template.hbs

```diff
@@ -0,0 +1,6 @@
+<p class="anomalies-table__values">
+  {{record.current}}/{{record.baseline}}
```

Comment:
are those numbers rounded? if not, perhaps use the d3 formatter? 
(some metrics have values in the millions+, and we don't want the text to overflow) 
https://github.com/linkedin/pinot/blob/master/thirdeye/thirdeye-frontend/app/utils/utils.js#L52

Reply:
Good call. Thanks.

Reply:
Yeah, I had issues with unusually long numbers here. Fixed them to 2 decimal places.